hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Listing.java

```diff
@@ -464,9 +466,10 @@ private boolean buildNextStatusBatch(S3ListResult objects) {
         // Skip over keys that are ourselves and old S3N _$folder$ files
```

Comment:
we should get ride of that $folder$ stuff these days, shouldn't we? not this PR though

Reply:
Yes, older S3N marker _$folder$  is not relevant anymore atleast for S3A.